embeddings/pipeline/hps_pipeline.py

```diff
@@ -120,15 +127,25 @@ def run(
         return study.trials_dataframe(), metadata
 
     def objective(self, trial: optuna.trial.Trial) -> float:
+        trial_name = standardize_name(f"study_{trial.study.study_name}_trial_{trial.number}")
+
         parameters = self.config_space.sample_parameters(trial=trial)
         parsed_params = self.config_space.parse_parameters(parameters)
-        args = self._get_evaluation_metadata(parsed_params)
-        pipeline = self.evaluation_pipeline(**args)
-        results = pipeline.run()
+        kwargs = self._get_evaluation_metadata(parsed_params, trial_name=trial_name)
+        pipeline = self._init_evaluation_pipeline(**kwargs)
+        results = pipeline.run(run_name=trial_name)
         metric = results[self.metric_name][self.metric_key]
         assert isinstance(metric, float)
         return metric
 
+    def _init_evaluation_pipeline(
```

Comment:
I'd suggest changing name of this method to _get_evaluation_pipeline(...) since I think the init here should rather be used to initialize objects and variables and assign them as class attributes instead of returning them;